### ISSUE TITLE
train at mpii,acc so small

### ISSUE THREAD

**Issue by mathpopo** [yuanyuanli85/Fast_Human_Pose_Estimation_Pytorch#16]
I try to run "python example/mpii.py -a hg --stacks 8 --blocks 1 --checkpoint checkpoint/hg_s8_b1/ " in your code , compare to your log.txt,loss can descend as same with you,but acc samll too mush,just "python example/mpii.py -a hg --stacks 8 --blocks 1 --checkpoint checkpoint/hg_s8_b1/ "  is whole?
![log](https://user-images.githubusercontent.com/21274466/66547260-6d5aa680-eb71-11e9-98ef-3e0b3b2dd8bc.png)

**Comment by yuanyuanli85** [comment]
looks like the train_loss does not drop too much.  Have you follow the instruction to disable cudnn for batchnorm layer if you are using pytorch0.4.x?

**Comment by mathpopo** [comment]
i use titan xp ,can run well,this effect is in the v100,the two graphics cards are different?

**Comment by mathpopo** [comment]
disable cudnn for batchnorm layer: sed -i "1194s/torch\.backends\.cudnn\.enabled/False/g"?i do this already,use pytorch 0.4.1,cuda 9.0

**Comment by mathpopo** [comment]
can you tell me your graphics cards hardware?

**Comment by mathpopo** [comment]
i use v100,resume from pre-trained model ,acc descend ,use titan xp , go 
![ee](https://user-images.githubusercontent.com/21274466/66636531-f7733f80-ec43-11e9-88f5-838fa618a4a5.png)
up

**Comment by yuanyuanli85** [comment]
The gpu I used is titan xp, not v100.  I guess some low-level difference b/w driver/cudnn caused the problem in v100.

**Comment by mathpopo** [comment]
thank you very mush.sorry ,other issue,if half body & side ,effect so bad,i descend threshold ,point error happen
![test1](https://user-images.githubusercontent.com/21274466/66694660-7752e580-ece8-11e9-8ea7-3f7b6459a828.png)

**Comment by mathpopo** [comment]
this is .0.1 threshold effect
![test2](https://user-images.githubusercontent.com/21274466/66695702-66f63700-ecf7-11e9-8e4c-c990af87e608.png)
